broad-wb-malkov.json

```diff
@@ -5,6 +5,22 @@
   "intent": "terra infrastructure",
   "production": false,
   "stability": "volatile",
+  "profile_vars": {
+    "thurloe-configs": {
+      "env": {
```

Comment:
A lot of these can be defaulted as @davidangb pointed out. Could we remove changes to this file from this PR and I'll add them in mine?

Reply:
OK, sounds good!